<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDlgClassClass</class>
 <widget class="QDialog" name="MessageDlgClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageDlgClass</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>btnOk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>btnNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>btnCancel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="memMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
